src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/View/Controller/TestCertificate/ExposureSubmissionTestCertificateViewModel.swift

```diff
@@ -0,0 +1,145 @@
+////
+// 🦠 Corona-Warn-App
+//
+
+import UIKit
+import OpenCombine
+
+class ExposureSubmissionTestCertificateViewModel {
+
+	// MARK: - Init
+
+	init(
+		testType: CoronaTestType = .pcr,
+		presentDisclaimer: @escaping () -> Void
+	) {
+		self.presentDisclaimer = presentDisclaimer
+		self.testType = testType
+		self.isPrimaryButtonEnabled = testType == .antigen
+	}
+
+	// MARK: - Overrides
+
+	// MARK: - Protocol <#Name#>
+
+	// MARK: - Public
+
+	// MARK: - Internal
+
+	let testType: CoronaTestType
+	private(set) var birthDayDate: String? {
+		didSet {
+			isPrimaryButtonEnabled = birthDayDate != nil
+		}
+	}
+
+	@OpenCombine.Published private(set) var isPrimaryButtonEnabled: Bool
+
+	enum ReuseIdentifiers: String, TableViewCellReuseIdentifiers {
+		case legalExtended = "DynamicLegalExtendedCell"
+		case birthdayDatePicker = "BirthdayDatePickerCell"
+	}
+
+	var dynamicTableViewModel: DynamicTableViewModel {
+		DynamicTableViewModel([
+			// Illustration with information text
+			.section(
+				header:
+					.image(
+						UIImage(
+							imageLiteralResourceName: "Illu_Test_Certificate"
+						),
+						accessibilityLabel: AppStrings.ExposureSubmission.TestCertificate.Info.imageDescription,
+						accessibilityIdentifier: AccessibilityIdentifiers.ExposureSubmission.TestCertificate.Info.imageDescription,
+						height: 161
+					),
+				cells: [
+					.body(
+						text: AppStrings.ExposureSubmission.TestCertificate.Info.body,
+						accessibilityIdentifier: AccessibilityIdentifiers.ExposureSubmission.TestCertificate.Info.body
+					),
+					testType == .antigen ? nil : .birthdayDatePicker(
+						placeholder: AppStrings.ExposureSubmission.TestCertificate.Info.birthDayPlaceholder,
+						accessibilityIdentifier: AccessibilityIdentifiers.ExposureSubmission.TestCertificate.Info.birthDayPlaceholder,
+						configure: { [weak self] _, cell, _ in
+							guard let birthdayDateInputCell = cell as? BirthdayDatePickerCell,
+								  let self = self else {
+								return
+							}
+							birthdayDateInputCell.$birthdayDate
+								.sink { newValue in
+									self.birthDayDate = newValue
+								}
+								.store(in: &self.subscriptions)
```

Comment:
I haven't tested it, but would the value entered in the cell survive a cell reuse if it was scrolled out of the screen?

Reply:
In fact it worked, by chance. I have revised and tested the reuse case. Good finding! 👍